software/spmd/bsg_cuda_lite_runtime/hardware_shared_mem_load_store/main.c

```diff
@@ -0,0 +1 @@
+../main/main.c
```

Comment:
We have a better solution than symlinking main.c -- we should just tell the compiler where main.o is

Reply:
This is an issue for another PR, all bsg_cuda_lite_runtime tests are symlinked in this way for the time being.

Reply:
@drichmond it's been a couple months but I agree with with @bornaehsani - let's fix that in another PR.

Reply:
Sure